scripts/create_test

```diff
@@ -257,7 +257,7 @@ OR
         args.no_batch = True
 
     if args.test_id is None:
-        args.test_id = CIME.utils.get_timestamp()
+        args.test_id = "%s.%s"%(CIME.utils.get_timestamp(), random.randrange(0,99,1))
```

Comment:
@jgfouca beat me to it. I was going to suggest this:

```python
def id_generator(size=6, chars=string.ascii_lowercase + string.digits):
    return ''.join(random.choice(chars) for _ in range(size))
```

(maybe we can put that in utils.py).

Then

id_generator()

generates a 6-character ID that has > 1 billion possibilities. To me that seems safe enough.

Credit: http://stackoverflow.com/questions/2257441/random-string-generation-with-upper-case-letters-and-digits-in-python